lib/fasterer/file_traverser.rb

```diff
@@ -13,7 +13,7 @@ class FileTraverser
 
     attr_reader :config
     attr_reader :parse_error_paths
-    attr_accessor :offenses_total_count
+    attr_reader :offenses_total_count
```

Comment:
if you remove the writer, how does line 64 work? `self.offenses_total_count += analyzer.errors.count` Is it not under test?

Reply:
@DamirSvrtan 	
You are right. it will create an error. I added a new commit.
Can you review?
Thank you.